### ISSUE TITLE
Test for extent should manage both integer and float

### ISSUE THREAD

**Issue by ThomasG77** [opengeospatial/ets-wfs30#56]
Running `mvn integration-test` by changing spatial extent values in https://github.com/opengeospatial/ets-wfs30/blob/master/src/test/resources/org/opengis/cite/wfs30/collections/collections.json#L47-L50 to integer lead to a cast issue in the tests when converting extent to bbox. Tests implementation for extent should support both integer and float IMO.
The return error (with cleaned output) is the following:
```
Tests in error: 
  JsonUtilsTest.testParseSpatialExtent:138 » ClassCast java.lang.Integer cannot ...
```

I've hit this issue while testing on my own implementation but this way was more simple to reproduce the issue.

**Comment by dstenger** [comment]
I agree, both, integer and float, should be supported.

**Comment by keshav-nangare** [comment]
Fixed with this PR #57.
